Add HSF History paper (#146)
HSF/documents: HSF-DOC/2022-01/hsf-history-paper.tex

\maketitle

\hypertarget{history}{%
\section{History}\label{history}}

Over the past 50 years, the experimental particle, nuclear and
astroparticle physics communities have iteratively evolved a significant
amount of community structure. This was a natural result of the growing
size, scale and time duration of the experiments and the centralisation
of facilities at large laboratories. National, and now international,
collaborations are typically required to build, operate and maintain the
large detectors used in these experiments. No single university or
laboratory can provide all of the necessary expertise and required
effort. The largest collaborations have grown from 100s of collaborators
in the 1990s to 1000s at (for example) the Large Hadron Collider (LHC)
at CERN. This community has also developed a broad ecosystem of
methodologies and technologies that are used to build successive
experiments and upgrades to existing experiments. While a specific
instrument can necessarily only be used for a single experiment at any
given time, the large commonalities in methodology should permit the

> development of research software which can be used widely in the

Reviewer: ```suggestion
development of research software that can be used widely in the
```

community for different projects. Despite this, much of the software
development remained somewhat siloed within individual experiments or,
at most, one or another host laboratory, with only a few exceptions.

> This is not to say that in the history of HEP there was no common software. 

Reviewer: ```suggestion
This is not to say that in the history of high-energy physics (HEP) there was no common software. 
```

individuals and solidified the active communities around the HSF. Once
the working groups had produced their domain-specific white papers, an
editorial team took charge of synthesising a unified single version of
the white paper, which was published with extremely wide community
support, 310 signing authors from 124 institutes~\cite{Albrecht2019}.

The CWP was not the only activity happening in the HSF at this time.
Particularly where there was an identified gap between different
communities, be these inter-experiment, between projects, or even
between theory and experiment, the HSF was ideally placed to bridge gaps
and bring different people together. Workshops on analysis ecosystems~\cite{ANALYSIS-ECO} 
and on computational aspects of event generators~\cite{COMP-GEN} were notable examples. In addition the HSF was a natural
forum for considering more radical software developments and their
potential, gathering experiment and expert feedback on progress and
possibilities~\cite{GEANTV-RD}.

\hypertarget{hsf-activities}{%
\section{HSF Activities}\label{hsf-activities}}

While much of the early HSF activity was focused on software community

> building through the CWP, working groups were started where common

Reviewer: ```suggestion
building through the CWP, working groups (WGs) were started where common
```

topics were obviously identified within the community, e.g., in the
domain of packaging and distributing HEP software stacks. These groups
brought together experts and interested parties and focused around
technical discussions.

In the wake of the CWP it was clear that this model would work very well
for the areas of most concern for the future, so the model was broadened

> and, over a few years, eight working groups were established with about

Reviewer: ```suggestion
and, over a few years, eight WGs were established with about
```

generators, led by the Generators working group~\cite{Valassi2021}.

In addition to being a focus for the exchange of ideas and techniques,
when WGs identify a concrete topic where a paper can usefully be
prepared, the HSF is a natural place for organising pan-experiment input
and encouraging some standardisation (e.g., in analysis level metadata
or in detector conditions data access). This has been recognised by more
formal bodies outside the HSF, such as WLCG and the LHCC, who often ask
the HSF to marshall community inputs for updates on development
activities and reviews of development plans. This is an important point
in terms of recognising the contribution of the organisation. In turn,
this fosters recognition for the contribution of our individual members
and helps their careers to advance.

This engagement with strategic bodies in the field, where the HSF
advocates for software investment~\cite{stewart_graeme_a_2018_2413005}, leads the HSF to
work in tandem with other funded software R\&D projects in HEP, where
the HSF will support funding applications and then work with these
projects, enhancing their connection to the community and the impact of

> their work. Practically projects can contribute through the working

Reviewer: ```suggestion
their work. Practically, projects can contribute through the working
```
